source/Playnite/Common/Extensions/StringExtensions.cs

```diff
@@ -86,6 +86,38 @@ public static string ToTileCase(this string source, CultureInfo culture = null)
             }
         }
 
+        public static bool IsStartOfStringAcronym(this string acronymStart, string input)
+        {
+            if (string.IsNullOrEmpty(input) || string.IsNullOrEmpty(acronymStart) || acronymStart.Length > input.Length)

```

Comment:
This should also exit if acronymStart is only one character long.

Reply:
Fixed in https://github.com/JosefNemec/Playnite/pull/3202/commits/f5f00f9913c5c6c57fbc6710f7ddfba519be2dce.

I couldn't find info about it but I think it makes sense to limit it to at least 2 since acronyms are words which requires having multiple characters at least.